### ISSUE TITLE
Can we work on stuff that is part of the public release milestone?

### ISSUE THREAD

**Issue by medensqui** [PlanHubMe/PlanHub#113]
Like easter eggs are fun and all, but we have a release goal of Thursday, that's 51 hours from now.

We need to get other things done that are more important

@thatoddmailbox @nywillb @willweinstein

**Comment by thatoddmailbox** [comment]
@medensqui That sounds smart! But I am also working on things people send feedback about. (weekend planner boxes, HW view bugginess) Quick and good responses to feedback makes happy users. Happy users tell their friends and are more likely to use PlanHub.

**Comment by medensqui** [comment]
@thatoddmailbox True! But once you're done with that, pls work on stuff that we need to do for public release! Remember easter eggs and new features are cool but we need to fix existing bugs, polish the webpage, and do all the things we said we needed for Day 1 first before we do things. Remember, CoursesPlus sometimes took 2 weeks to update things but we told people we were working on it and they were fine with it. So just pointing it out

**Comment by thatoddmailbox** [comment]
@medensqui I know, but the center of all your planning stuff is more important than a bug on a page on Courses no one really uses.

(if it makes you feel any better, right now I'm working on #13)

**Comment by medensqui** [comment]
@thatoddmailbox True. YAY
But def focus on feedback bugs esp. weinstein's cause they sound very undesireable
Also, I meant the bugs and stuff more than My Day

My Day is preferably ready Day 1 but not a priority

**Comment by thatoddmailbox** [comment]
@medensqui `!false`.

Yeah @willweinstein appears to be experiencing some fun bugs. I sent a followup comment about it, but no response yet...

**Comment by medensqui** [comment]
@thatoddmailbox False?

Yeah...

**Comment by thatoddmailbox** [comment]
@medensqui NO

`!false`

DO YOU NOT SEE THE EXCLAMATION

IT MEANS THE OPPOSITE OF

**Comment by medensqui** [comment]
@thatoddmailbox AH OK

WHY ARE WE USING CAPS

**Comment by thatoddmailbox** [comment]
@medensqui IDK I FELT LIKE YELLING ok I'll stop now.

**Comment by thatoddmailbox** [comment]
@medensqui Should I attempt a #13 again?

**Comment by medensqui** [comment]
@thatoddmailbox Umm...It's something that needs to get fixed eventually. So I guess yeah!

**Comment by thatoddmailbox** [comment]
@medensqui UGH I BROKE SOMETHING AND NOW GITHUB WON'T AUTOMATICALLY MERGE

**Comment by medensqui** [comment]
@thatoddmailbox :tongue:

**Comment by medensqui** [comment]
@thatoddmailbox HAHAHAHAHHA

**Comment by thatoddmailbox** [comment]
@medensqui HA FIXED IT

**Comment by thatoddmailbox** [comment]
@medensqui I think a smart idea would be to focus on bug fixes, polish, etc. and leave My Day for after the deadline. We only get one chance at a first impression, so it better be good! We could leave it in its current state (homepage & feature enable say "coming soon") and maybe have some super-secret beta version that is in early development, but I think that if we try to rush it for Thursday, it won't be very good.

?

**Comment by medensqui** [comment]
@thatoddmailbox True!

**Comment by medensqui** [comment]
@thatoddmailbox @nywillb @willweinstein CAN I JUST POINT OUT THAT PUBLIC RELEASE IS TOMMOROW! YES I SAID TOMMOROW! AND WE HAVE DONE 50% OF PUBLIC RELEASE STUFF. LETS FINISH IT

**Comment by thatoddmailbox** [comment]
@medensqui CALM YOURSELF AAAAAGGGGHH

Also we can maybe push it to Friday.

Also I have to do #14 tonight so it doesn't break the live site.

**Comment by medensqui** [comment]
@thatoddmailbox ok, focus on bugs. but it's really important. Ok
STOP WORRYING ABOUT STUDERPLANNER.TK AND WORRY ABOUT THE FRIKIN PUBLIC RELEASE

THERES NO POINT IN FIXING FRIKIN MINOR BUGS WHEN THERES NO PRODUCT

**Comment by thatoddmailbox** [comment]
@medensqui GEEZ OK THE DOMAINS TOOK LIKE 2 MINUTES.

ALSO "ONE ROW STOPPED BEING REGISTERED IN HW VIEW" IS NOT MINOR

**Comment by thatoddmailbox** [comment]
THE PUBLIC RELEASE MILESTONE IS FINISHED YAY

**Comment by medensqui** [comment]
@thatoddmailbox YAY SO NOW LET'S START OUR OTHER MILESTONE. AHEM AHEM COUGH MY DAY AHEM AHEM

**Comment by thatoddmailbox** [comment]
@medensqui yes i know but @c20et wants to submit it for the hackathon and we can't start it yet if we do.

idk if it is a good idea to submit it there, but...

**Comment by medensqui** [comment]
@thatoddmailbox _Meh_

Idk...

**Comment by thatoddmailbox** [comment]
@medensqui also even if we don't start my day until we are allowed to, they still would not like the base of planhub

**Comment by thatoddmailbox** [comment]
@medensqui we should actually maybe figure out if that's what we're gonna do

**Comment by thatoddmailbox** [comment]
@medensqui ?

**Comment by medensqui** [comment]
@thatoddmailbox yeah...

**Comment by thatoddmailbox** [comment]
@medensqui ??
